bedwars-api/src/main/java/com/tomkeuper/bedwars/api/configuration/ConfigPath.java

```diff
@@ -121,6 +121,7 @@ public class ConfigPath {
     public static final String GENERAL_CONFIGURATION_ARENA_GROUPS = "arenaGroups";
     public static final String GENERAL_CONFIGURATION_REJOIN_TIME = "rejoin-time";
     public static final String GENERAL_CONFIGURATION_RE_SPAWN_INVULNERABILITY = "re-spawn-invulnerability";
+    public static final String GENERAL_CONFIGURATION_ANNOUNCEMENT_SEND_EVERY = "announcement-send-every";
```

Comment:
idk here but maybe use that: `in-game-announcement-cooldown` and `GENERAL_CONFIGURATION_IN_GAME_ANNOUNCEMENT_COOLDOW`

Reply:
> idk here but maybe use that: `in-game-announcement-cooldown` and `GENERAL_CONFIGURATION_IN_GAME_ANNOUNCEMENT_COOLDOW`

Okay. I will change everything

Reply:
> > idk here but maybe use that: `in-game-announcement-cooldown` and `GENERAL_CONFIGURATION_IN_GAME_ANNOUNCEMENT_COOLDOW`
> 
> Okay. I will change everything

done changing everything